diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1db6a5e1..30243993 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,6 +29,7 @@ option(LSL_BUNDLED_BOOST "Use the bundled Boost by default" ON)
 option(LSL_BUNDLED_PUGIXML "Use the bundled pugixml by default" ON)
 option(LSL_SLIMARCHIVE "Use experimental but smaller serialization code" OFF)
 option(LSL_TOOLS "Build some experimental tools for in-depth tests" OFF)
+option(LSL_SKIP_INSTALL "Skip install() rules" OFF)
 
 mark_as_advanced(LSL_SLIMARCHIVE LSL_FORCE_FANCY_LIBNAME)
 
@@ -296,6 +297,7 @@ if(LSL_BUILD_STATIC)
 	list(APPEND LSLTargets lslobj lslboost)
 endif()
 
+if(NOT LSL_SKIP_INSTALL)
 install(TARGETS ${LSLTargets}
 	EXPORT LSLTargets
 	COMPONENT liblsl
@@ -357,3 +359,4 @@ if(LSL_BUILD_EXAMPLES)
 endif()
 
 LSLGenerateCPackConfig()
+endif()
